source/lib-intro.tex

```diff
@@ -1891,9 +1891,9 @@
   \tcode{X::const_pointer}                &
   \tcode{static_cast<} \tcode{X::const_pointer\brk{}>(x) == q} & \\ \rowsep
 
-\tcode{pointer_traits<\brk{}X::pointer\brk{}>::pointer_to(r)}  &
+\tcode{pointer_traits<\brk{}X::pointer\brk{}>::pointer_to(*p)}  &
```

Comment:
I'm a little worried that this can be interpreted as *only* applying to a construct of this form (ie, not applying to the case of `pointer_traits<X::pointer>::pointer_to(r)` for an arbitrary lvalue `r` of the right type). Eg, when we place requirements on `*r++` in the input iterator requirements, the user is not permitted to draw conclusions from that about the expression `r++` (without the `*`).

Reply:
OK, we could introduce `r` before the table, as being the object that `*p` yields, so we can keep the requirement that `pointer_to(r) == p`.

Reply:
Or I could create an LWG issue to require the `pointer_to(r) == p` equialty if you're worried it's novel.

Reply:
d2afc9a9b06e8fc495d94506620e54c0b62b36fc shows the original definition of `r` that was removed.

Reply:
I see. So even the old formulation would have carried the implication that only `*p` can be used here. Let's reintroduce `r` to retain the modicum of indirection between the syntax here and the `*p`.